other/ner_fuzzy_manual.py

```diff
@@ -0,0 +1,126 @@
+from typing import List, Optional
+import copy
+from collections import defaultdict
+import prodigy
+from prodigy.components.loaders import JSONL
+from prodigy.components.preprocess import add_tokens
+from prodigy.util import split_string, set_hashes
+import spacy
+from spacy.tokens import Span
+from spaczz.matcher import FuzzyMatcher
+
+
+def parse_phrase_patterns(patterns):
```

Comment:
I think this is a great simplification 👍

Reply:
Thanks :)